templates/guests/redhat/network_route.erb

```diff
@@ -0,0 +1,7 @@
+#VAGRANT-BEGIN
+# The contents below are automatically generated by Vagrant. Do not modify.
+# <%= routes.inspect %>
+<% routes.split(';').each do |route| %>
```

Comment:
Is this `routes` option coming from a Vagrantfile? What if someone uses something other than `;` to split their IPs? If I remember right this `network` option from `configure_networks` comes from `config.vm.network :network_type, routes: "routes;go;here"` which means someone could split with commas or spaces and this won't work :thinking: